config/environments/development.rb

```diff
@@ -14,18 +14,19 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  config.cache_store = :redis_cache_store, { url: ENV["REDIS_URL"] }
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join('tmp', 'caching-dev.txt').exist?
-    config.cache_store = :memory_store
```

Comment:
Why blow this entire thing away? You just need to update this line to get `redis_cache_store` to work, right?
Or is it because of this dependance on `tmp/caching-dev.txt` ? (what is that file? is it generated by rails?)

Reply:
Yeah -- I have no idea what this is for. I think it might be for local-session caching via the file.